ix-dev/community/windmill/templates/docker-compose.yaml

```diff
@@ -0,0 +1,176 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% if values.windmill.web_host.startswith("https://") or values.windmill.web_host.startswith("http://") %}
+  {% do tpl.funcs.fail("Expected [windmill.web_host] to not start with [http(s)://]") %}
+{% endif %}
+{% if values.windmill.web_host.endswith("/") %}
+  {% do tpl.funcs.fail("Expected [windmill.web_host] to not end with [/]") %}
+{% endif %}
+{% if ":" in values.windmill.web_host %}
+  {% do tpl.funcs.fail("Expected [windmill.web_host] to not contain port [:###]") %}
+{% endif %}
+
+{% set perm_container = tpl.deps.perms(values.consts.perms_container_name) %}
```

Comment:
Add another line here like
{% set perms_config = ... %}

And use it on add_or_skip_action (see other apps for examples)